packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1166,6 +1173,13 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return this.context.configuration;
     }
 
+    public formRequestSummarizerFn(requestOptions: ISummarizerRequestOptions) {
```

Comment:
Does it need to be public? What is the usage pattern we expect? having some documentation above public API would be nice

Reply:
There may not be a real-world use case. My usage is in a test. I wanted to be able to create a Summarizer (and summarizer client) given a Container or ContainerRuntime. I couldn't find a way to do that without either exposing ContainerRuntime.loader or putting this API on ContainerRuntime. I actually want to document a few entry points that I added.